### PR TITLE
Task about

### PR DESCRIPTION
**Pull request by Steph-ski**
-About section HTML & CSS
- Write About section text

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -32,6 +32,16 @@ <h1>I'm Steph, a Junior Software Developer</h1>
 	</section>
```
Comment (barebarebears) [inline index.html:32]: this appears to be a stray closing tag for a section that is no longer there? Or perhaps mislabelled closing tag for the nav?

File: index.html [hunk 2]
```diff
@@ -11,7 +11,7 @@
 <body>
 	<nav class="nav-links">
```
Comment (barebarebears) [inline index.html:12]: I can't see a closing tag for the <nav> (although it still appears to be functioning properly so have I missed something?)

File: index.html [hunk 3]
```diff
@@ -32,6 +32,16 @@ <h1>I'm Steph, a Junior Software Developer</h1>
 	</section>
 </header>
```
Comment (barebarebears) [inline index.html:33]: this also looks like a stray closing tag. I can't find an opening tag for header (although can see 'head' further up)

**Review by nesclark — CHANGES_REQUESTED** [review]
The about me section looks okay but there are a few issues now with the original code further up. I'm not sure how all this is impacting on your CSS but interestingly your page functions perfectly well so I'll admit I'm confused!

**Review by nesclark — APPROVED** [review]

**Pull request merged**